task/src/task.cpp

```diff
@@ -1,7 +1,12 @@
 #include "lute/task.h"
 
+#include "lua.h"
+#include "lute/ref.h"
 #include "lute/runtime.h"
 
+// taken from extern/luau/VM/lcorolib.cpp
+static const char* const statnames[] = {"running", "suspended", "normal", "dead", "dead"};
```

Comment:
Is this not exposed? If so, is there an API exposed already for getting the name for errors? cc @vegorov-rbx